project.clj

```diff
@@ -5,7 +5,7 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [org.clojure/clojurescript "0.0-3269"]
+                 [org.clojure/clojurescript "1.7.107"]
```

Comment:
I upgraded just to stay current, but my changes don't depend on this newer version so you can roll this back if you want.